src/cmd/deploy.go

```diff
@@ -101,23 +102,57 @@ func init() {
 
 	deployCreateCmd.Flags().String("commit-sha", "", "git commit sha associated with the event (OPSLEVEL_DEPLOYER_NAME)")
 	deployCreateCmd.Flags().String("commit-message", "", "git commit message associated with the event (OPSLEVEL_DEPLOYER_EMAIL)")
-	viper.BindPFlags(deployCreateCmd.Flags())
-	viper.BindEnv("integration-url", "OPSLEVEL_INTEGRATION_URL", "OL_INTEGRATION_URL")
-	viper.BindEnv("dry-run", "OPSLEVEL_DRY_RUN", "OL_DRY_RUN")
-	viper.BindEnv("git-path", "OPSLEVEL_GIT_PATH", "OL_GIT_PATH")
-	viper.BindEnv("deploy-number", "OPSLEVEL_DEPLOY_NUMBER", "OL_DEPLOY_NUMBER")
-	viper.BindEnv("deploy-url", "OPSLEVEL_DEPLOY_URL", "OL_DEPLOY_URL")
-	viper.BindEnv("dedup-id", "OPSLEVEL_DEDUP_ID", "OL_DEDUP_ID")
-	viper.BindEnv("deployer-name", "OPSLEVEL_DEPLOYER_NAME", "OL_DEPLOYER_NAME")
-	viper.BindEnv("deployer-email", "OPSLEVEL_DEPLOYER_EMAIL", "OL_DEPLOYER_EMAIL")
-	viper.BindEnv("commit-sha", "OPSLEVEL_COMMIT_SHA", "OL_COMMIT_SHA")
-	viper.BindEnv("commit-message", "OPSLEVEL_COMMIT_MESSAGE", "OL_COMMIT_MESSAGE")
+	if err := viper.BindPFlags(deployCreateCmd.Flags()); err != nil {
```

Comment:
hmm - do we know when this binding of flags would fail?  I'm concerned this add a TON of boilerplate for no real value.  If this is worth it then it would make make sense to make a wrapper method to handle the error checking

```
func BindEnv(key string, envs string...) {
	cobra.CheckErr(viper.BindEnv(key, ...envs))
}
```

Reply:
Agreed with cutting down the redundancy. I plan on looking at this again in a follow up PR and likely writing a wrapper function like your example